src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
 // write code here
+
+const employeeList = document.querySelector('ul');
+
+function sortList(list) {
+  const items = Array.from(list.children);
+
+  items.sort((a, b) => {
+    const salaryA = parseSalary(a.getAttribute('data-salary'));
+    const salaryB = parseSalary(b.getAttribute('data-salary'));
+
+    return salaryB - salaryA;
+  });
+
+  items.forEach((item) => list.appendChild(item));
+}
+
+function parseSalary(salaryStr) {
+  return parseInt(salaryStr.replace(/[^0-9.-]+/g, ''));
```

Comment:
The `parseSalary` function uses `parseInt`, which will only parse the integer part of the salary. If the salary includes decimal values, consider using `parseFloat` instead to ensure accurate parsing of the salary values.